test_conformance/api/test_mem_object_properties_queries.cpp

```diff
@@ -0,0 +1,317 @@
+//
+// Copyright (c) 2020 The Khronos Group Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include "testBase.h"
+#include "harness/typeWrappers.h"
+#include <vector>
+#include <algorithm>
+
+using namespace std;
```

Comment:
Very minor thing, but since you have made use of the namespace resolution `std::` for `std::vector` and `std::string` in this file, would it make sense to remove this using statement?